packages/expo-cli/src/commands/eas-build/utils/expoUpdates/index.ts

```diff
@@ -0,0 +1,2 @@
+export { configureUpdatesAndroidAsync, setUpdatesVersionsAndroidAsync } from './android';
```

Comment:
In last comment, I suggested removing that file and import directly from `android` and `ios`, but this is also ok